CMakeLists.txt

```diff
@@ -45,7 +45,14 @@ FetchContent_Declare(
 # Enable legacy mode for CPP_BINDGEN for users of GT 1.0
 # TODO remove in GT 2.0
 set(CPP_BINDGEN_GT_LEGACY ON)
+if (GT_USE_CLANG_CUDA)
+    set(build_testing_ ${BUILD_TESTING})
+    set(BUILD_TESTING OFF)
+endif()
 FetchContent_MakeAvailable(cpp_bindgen)
+if (GT_USE_CLANG_CUDA)
+    set(BUILD_TESTING ${build_testing_})
+endif()
```

Comment:
Cpp_bindgen’s CMake setup does not yet support Clang-CUDA, sot its unit tests are disabled (the tests inside GridTools using cpp_bindgen are still enabled, as they work perfectly).